lib/Field/stories/FieldStory.js

```diff
@@ -76,6 +76,176 @@ storiesOf('Components', module).add('Field', () => {
         title="FieldNew"
         description="A newer, Field component, can contain Content and Meta"
       >
+        <FieldNew
+          className={cx({ 'field-active': status === ACTIVE }, 'mb-0')}
+          dir={dir}
+          inStructureEditMode={inStructureEditMode}
+        >
+          <FieldNew.Meta>
+            {isRepeatable && (
+              <FieldNew.RepeatableControls
+                repeatPosition={0}
+                isLastRepeat={false}
+                isActive={status === ACTIVE}
+              />
+            )}
+            <FieldNew.Label
+              className={cx({
+                'text-right': dir === 'ltr',
+                'text-left': dir === 'rtl',
+                'pt-6': !isRepeatable
+              })}
+            >
+              {label}
+            </FieldNew.Label>
+            <FieldNew.Validations
+              validations={mockValidations}
+              fieldId="123"
+              className="flex-col items-end"
+            />
+            <FieldNew.Actions
+              actions={mockActions}
+              fieldId="123"
+              className="items-end"
+            />
+          </FieldNew.Meta>
+          <FieldNew.Content
+            instructions={
+              <FieldNew.Instructions>{instructions}</FieldNew.Instructions>
+            }
+            status={status}
+          >
+            {!inStructureEditMode ? (
+              <div>
+                <h1>Heading content</h1>
+                <p>
+                  The fossil record indicates that birds evolved from feathered
+                  ancestors within the theropod group, which are traditionally
+                  placed within the saurischian dinosaurs, though a 2017
+                  paper[4] has put them in a proposed clade Ornithoscelida,
+                  along with the Ornithischia.
+                </p>
+                <table>
+                  <tbody>
+                    <tr>
+                      <td>
+                        <p>
+                          <strong>Heading 1</strong>
+                        </p>
+                      </td>
+                      <td>
+                        <p>
+                          <strong>Heading 2</strong>
+                        </p>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>
+                        <p>Cell 1</p>
+                      </td>
+                      <td>
+                        <p>Cell 2</p>
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+            ) : (
+              <div className="p-3" />
+            )}
+          </FieldNew.Content>
+          {isRepeatable && (
+            <FieldNew.RepeatButton
```

Comment:
With reference to my comment on splitting this component, this would make this more readable.

Currently I'm reading it, thinking "Ok we have a button here" - when in reality we are actually only rendering the divider. This is a mental model we don't have to investigate if instead we have here `FieldNew.ConnectorLine`